documentation/configuration.md

```diff
@@ -393,3 +393,239 @@ In addition to enabling custom providers, `dotnet monitor` also allows you to di
+| `ExcludePaths` | string[] | false | The list of request path patterns to ignore. Request paths matching a pattern in this list will be ignored. | `null` | | |
+
+The `IncludePaths` and `ExcludePaths` support [wildcards and globbing](#aspnet-request-path-wildcards-and-globbing).
+
+##### Example
+
+Usage that is satisfied when 10 requests take longer than 3 seconds during a 1 minute period for all paths under the `/api` route:
+
+```json
+{
+  "RequestCount": 10,
+  "RequestDuration": "00:00:03",
+  "SlidingWindowDuration": "00:01:00",
+  "IncludePaths": [ "/api/**/*" ]
+}
+```
+
+#### `AspNetResponseStatus` Trigger
+
+A trigger that has its condition satisfied when the number of HTTP responses that have status codes matching the pattern list is above the specified threshold for a sliding window of time. The request paths can be filtered according to the described patterns.
+
+##### Properties
+
+| Name | Type | Required | Description | Default Value | Min Value | Max Value |
+|---|---|---|---|---|---|---|
+| `StatusCodes` | string[] | true | The list of HTTP response status codes to monitor. Each item of the list can be a single code or a range of codes (e.g. `"400-499"`). | | | |
+| `RequestCount` | int | true | The threshold number of responses with matching status codes. | | | |
+| `SlidingWindowDuration` | TimeSpan? | false | The sliding time window in which the number of responses with matching status codes must occur. | `"00:01:00"` (one minute) | `"1.00:00:00"` (1 day) | `"00:00:01"` (one second) |
+| `IncludePaths` | string[] | false | The list of request path patterns to monitor. If not specified, all request paths are considered. If specified, only request paths matching one of the patterns in this list will be considered. Request paths matching a pattern in the `ExcludePaths` list will be ignored. | `null` | | |
+| `ExcludePaths` | string[] | false | The list of request path patterns to ignore. Request paths matching a pattern in this list will be ignored. | `null` | | |
+
+The `IncludePaths` and `ExcludePaths` support [wildcards and globbing](#aspnet-request-path-wildcards-and-globbing).
+
+##### Example
+
+Usage that is satisfied when 10 requests respond with a 5XX status code during a 1 minute period for all paths under the `/api` route:
+
+```json
+{
+  "StatusCodes": [ "500-599" ],
+  "RequestCount": 10,
+  "SlidingWindowDuration": "00:01:00",
+  "IncludePaths": [ "/api/**/*" ]
+}
+```
+
+#### `EventCounter` Trigger
+
+A trigger that has its condition satisfied when the value of a counter falls above, below, or between the described threshold values for a duration of time.
+
+##### Properties
+
+| Name | Type | Required | Description | Default Value | Min Value | Max Value |
```

Comment:
Not necessarily here, but we should probably have a link to the well-known .net counters docs.